### PR TITLE
feat(useSafeArea): new function

### PR DESCRIPTION
**Pull request by okxiaoliang4**
get safe area in js

```ts
import { useSafeArea } from '@vueuse/core'

const { top, right, bottom, left } = useSafeArea()
```

### CODE REVIEW

**Comment by wheatjs** [comment]
Can you give a better description of what this does?

**Comment by okxiaoliang4** [comment]
@wheatjs 
> Can you give a better description of what this does?

Reactive `env(safe-area-inset-*)`

image

File: packages/core/useSafeArea/index.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { ref } from 'vue-demi'
+import { isClient, useCssVar, useDebounceFn, useEventListener } from '@vueuse/core'
+
+const topVarName = '--vue-use-safe-area-top'
```
Comment (wheatjs) [inline packages/core/useSafeArea/index.ts:4]: I think `vueuse` here should be a single word, so `--vueuse-safe-area-top` instead

Comment (okxiaoliang4) [reply packages/core/useSafeArea/index.ts:4]: ok~

**Comment by antfu** [comment]
How about name it `useScreenSafeArea` to be more clear?

**Comment by okxiaoliang4** [comment]
ok~

**Pull request merged**